docs/UserGuide.adoc

```diff
@@ -12,7 +12,33 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/se-edu/addressbook-level4
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `W-11 B4`      Since: `Sep 2017`      Licence: `MIT`
+
+== Welcome to Unify
+Unify is a Command Line Interface AddressBook that helps manage your contacts efficiently.
+
+=== Do More with Less
+Unlike conventional Address Books, Unify adopts a Command Line Interface or, CLI.
```

Comment:
Unlike conventional Address Books, Unify adopts CLI (Command Line Interface) for its user's interaction.